src/Webapi/Webapi__Fetch.res

```diff
@@ -6,6 +6,7 @@ type body
 type bodyInit
 type headers
 type headersInit
+type responseInit
```

Comment:
I wasn't sure if I should add alias the types from the top level or not as it says they are here for compatibility purposes. Let me know if I should remove them 👍

Reply:
This is fine. We can re-evaluate these for the eventual rewrite of the bindings.